sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClient.java

```diff
@@ -185,6 +188,7 @@ public String getEntityPath() {
      * @throws NullPointerException if {@code batch}, {@code transactionContext} or
      * {@code transactionContext.transactionID} is {@code null}.
      * @throws AmqpException if {@code messages} is larger than the maximum allowed size of a single batch.
```

Comment:
Don't we wrap this AmqpException with ServiceBusException for this error?

Reply:
yes we map it, let me change this to  SBE.